.github/workflows/ci.yml

```diff
@@ -1,14 +1,24 @@
 name: PHPUnit
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
-
+    name: Run tests on ${{ matrix.php }}
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        php: [ '7.4', '8.0', '8.1', '8.2' ]
+
     steps:
-      - uses: actions/checkout@v1
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
```

Comment:
Hi @theodorejb is this the correct pointer to the right fork? Wondering if this should point to the main fork?

Reply:
Gotcha, thanks for the clarification!